Assets/Scripts/Minigame/DragBlanket.cs

```diff
@@ -104,16 +131,29 @@ private void OnMouseDrag()
                 _rigidbody2D.position = _pos;
             }
 
-            if (_distance >= _winDistance && _counter == 3)
+            if (_standingUp)
             {
-                _tmpUGUI.text = "" + _winMessage;
-                IsDone = true;
+                if (_distance >= _winDistance && _counter == 3)
+                {
+                    _tmpUGUI.text = "" + _winMessage;
+                    IsDone = true;
+                }
+            }
+            else if(!_standingUp)

```

Comment:
if(bool)
else if(!bool) //this if is unnecessary. a bool only has two states, if it's true, the first if will execute, otherwise the else block will